lib/queries/getApplication.ts

```diff
@@ -0,0 +1,32 @@
+import { FetchResult } from "apollo-link";
+import gql from "graphql-tag";
+
+import { fetchManagement } from "../../src/utils/fetchManagement";
+import type { ProviderApplication } from "@lib/@types/Application";
+
+const GET_APPLICATION_QUERY = gql`
+  query getApplicationQuery($id: String!) {
+    provider {
+      application(filters: { id: $id }) {
+        id
```

Comment:
Do you think it's a good idea to return the id?

Reply:
I did it that way because I use the `getApplication` response to update the application, and `updateApplication` needs the Application id. But, I think I could add that id myself to the App object.

Reply:
I'm worried that this sensible data could be catch in clear if we do have it in the response. Am I wrong @Fenntasy?

Reply:
hmmm I don't really see the ID as a sensitive data, especially if we only use this query in a script that run on the CI.

But what strikes me as weird is to have this code in the `lib` part of Account when it's (I suppose?) only used in a script. I don't think we should, in the application do those kind of request, would we?

Reply:
Where do you think we should put these scripts then?

Couple questions to clarify. Do we want to permit Clients (from clients, not us) to create/delete applications? If we do, why couldn't it be in the NodeSDK?
And if we don't, where should we put this?

Reply:
that's a good point, my mistake 😅